File: Java/AutomationFramework/src/com/app/BDDTests/LoginPageTests/step_definitions/LoginPageSteps/LoginLogoTest.java
package com.app.BDDTests.LoginPageTests.step_definitions.LoginPageSteps;

import com.app.pages.LoginPage;
import io.cucumber.java.en.Given;
import io.cucumber.java.en.Then;
import io.cucumber.java.en.When;
import org.testng.Assert;

public class LoginLogoTest {
    LoginPage Login_Page;
    boolean isLogoPresent;
    @Given("I navigated to OrangeHRM portal")
    public void iNavigatedToOrangeHRMPortal() {
        Login_Page = LoginPage.GetLoginPage()
                .Init_Elements();
    }

    @When("Check for the logo image")
    public void checkForTheLogoImage() {
        isLogoPresent = Login_Page.IsLogoPresent();
    }

    @Then("The logo should be present")
    public void theLogoShouldBePresent() {
        Assert.assertEquals(isLogoPresent, true);
    }
}

File: DotNet/Framework/AutomationFramework/Tests.BDD/StepDefinitions/LoginPageSteps/LoginLogoTest.cs
using NUnit.Framework;
using Pages;

namespace Tests.BDD.StepDefinitions.LoginPageSteps
{
    [Binding]
    public class LoginLogoTest
    {
        LoginPage Login_Page;
        bool isLogoPresent;

        [Given(@"I navigated to OrangeHRM portal")]
        public void GivenINavigatedToOrangeHRMPortal()
        {
            Login_Page = LoginPage.Login_Page;
        }

        [When(@"Check for the logo image")]
        public void WhenILookForTheLogoImage()
        {
            Login_Page.IsLogoPresent(out isLogoPresent);
        }

        [Then(@"The logo should be present")]
        public void ThenTheLogoShouldBePresent()
        {
            Assert.That(isLogoPresent, Is.EqualTo(true));
        }

    }
}
